tests/test_filterset.py

```diff
@@ -718,6 +718,27 @@ class Meta:
         self.assertEqual(f.qs.count(), 1)
         self.assertQuerysetEqual(f.qs, [self.alex.pk], lambda o: o.pk)
 
+    def test_fields_set_with_dict(self):
+        class F(FilterSet):
+            class Meta:
+                model = User
+                fields = ['username', 'status', 'is_active', 'first_name']
+                together = [
+                    ({'username': ['status']}),
+                    ({'first_name': ['is_active']}),
+                ]
+
+        f = F({}, queryset=self.qs)
+        self.assertEqual(f.qs.count(), 2)
+        f = F({'username': 'alex'}, queryset=self.qs)
+        self.assertEqual(f.qs.count(), 0)
+        f = F({'username': 'alex', 'status': 1}, queryset=self.qs)
+        self.assertEqual(f.qs.count(), 1)
+        self.assertQuerysetEqual(f.qs, [self.alex.pk], lambda o: o.pk)
+        f = F({'status': 1}, queryset=self.qs)
+        self.assertEqual(f.qs.count(), 1)
+        self.assertQuerysetEqual(f.qs, [self.alex.pk], lambda o: o.pk)
+
```

Comment:
Would be nice if this would work with an OrderingFilter, too.

The following does not fail with this PR:
```diff
diff --git i/tests/test_filterset.py w/tests/test_filterset.py
index 62afff3..edd3f55 100644
--- i/tests/test_filterset.py
+++ w/tests/test_filterset.py
@@ -17,6 +17,7 @@
 from django_filters.filters import ChoiceFilter
 from django_filters.filters import ModelChoiceFilter
 from django_filters.filters import ModelMultipleChoiceFilter
+from django_filters.filters import OrderingFilter
 from django_filters.filters import UUIDFilter
 from django_filters.filters import BaseInFilter
 from django_filters.filters import BaseRangeFilter
@@ -720,6 +721,8 @@ class Meta:
 
     def test_fields_set_with_dict(self):
         class F(FilterSet):
+            o = OrderingFilter(fields=('username',))
+
             class Meta:
                 model = User
                 fields = ['username', 'status', 'is_active', 'first_name']
@@ -739,6 +743,9 @@ class Meta:
         self.assertEqual(f.qs.count(), 1)
         self.assertQuerysetEqual(f.qs, [self.alex.pk], lambda o: o.pk)
 
+        f = F({'o': 'username'}, queryset=self.qs)
+        self.assertEqual(f.qs.count(), 0)
+
 
 # test filter.method here, as it depends on its parent FilterSet
 class FilterMethodTests(TestCase):
```